src/main/java/org/opendevstack/provision/controller/DefaultController.java

```diff
@@ -86,7 +98,23 @@ String home(Model model) {
   }
 
     @RequestMapping("/provision")
-    String provisionProject(Model model, Authentication authentication, @CookieValue(value = "crowd.token_key", required = false) String crowdCookie){
+    String provisionProject(Model model, Authentication authentication, @CookieValue(value = "crowd.token_key", required = false) String crowdCookie, HttpServletRequest request)
+    {
+    	if (crowdCookie == null && request != null) {
+    		logger.debug("cookie NULL + " + request);
+    		
+    		Cookie[] cookies = request.getCookies();
+    		for (Cookie cookie : cookies) 
+    		{
+        		logger.debug("cookie" + cookie.getName());
+    			
+    			if (cookie.getName().equals(this.crowdCookieKey)) 
+    			{
+    				crowdCookie = cookie.getValue();
```

Comment:
currently crowdCookie is a method variable which is not assigned to an instance variable or used after this line whithin the method... is this on purpouse? Or the method should fail if the cookie is not set?